CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 
 
+## Unreleased
+
+### Deprecated
+
+- The `debug_plugins` option for `PluginClient` is deprecated and will be removed in 2.0. Use the decorator design pattern instead like in https://github.com/php-http/HttplugBundle/blob/master/Collector/ProfilePlugin.php.
```

Comment:
Maybe use a markdown link here instead of the whole URL. Also, this is not a permanent link. A link with a commit hash in it might be better.